File: pset6/mariopy/mario.py
from cs50 import get_int

# Gets input from user and makes sure it is between 1 to 8.
while True:
    num = get_int("Height: ")
    if num >= 1 and num <= 8:
        break

# Prints the triangle.
for i in range(1, num + 1):
    for j in range(1, num + 1):
        if(j <= num - i):
            print(' ', end='')
        else:
            print('#', end='')
    print()

File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Asks the user for block height
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);
    
    // Prints the block
    for (int x = 1; x <= height; x++)
    {
        for (int y = x; y < height; y++)
        {
            printf(" ");
        }
        for (int z = 1; z <= x; z++)
        {
            printf("#");
        }
        printf("\n");
    }
}